Formula/tinyscheme.rb

```diff
@@ -20,35 +20,38 @@ def install
     share.install("init.scm")
     bin.install("scheme")
   end
+
+  test do
+    assert_match "Usage: tinyscheme", shell_output("#{bin}/scheme -\?", 1)
+  end
 end
 
 __END__
---- a/makefile  2011-01-16 20:51:17.000000000 +1300
-+++ b/makefile  2012-04-08 22:38:11.000000000 +1200
+--- a/makefile
++++ b/makefile
```

Comment:
@woodruffw what really would be nice if you documented why each of those variables is set as it is and which ones are set as the makefile says they should be for OS X and which ones you needed to change, regardless of whether the final outcome is a patch, heredoc, directly running the commands, whatever.

The one gotcha is that you will need to add  ENV.refurbish_args to the formula if you run the commands inline. When you call "make", superenv does that automatically, but that is not true for inline commands unless you add  ENV.refurbish_args.